File: pinyougou_seckill_web/src/main/webapp/js/service/seckillService.js
app.service("seckillService",function ($http) {

    this.findSeckillGoods=function(){
        return $http.get("seckill/displaySeckillGoods.do");
    }
    this.findOne=function(seckillGoodsId){
        return $http.get("seckill/findOne.do?seckillGoodsId="+seckillGoodsId);
    }
    //抢单:
    this.createOrder=function(seckillGoodsId){
        return $http.get("seckill/createOrder.do?seckillGoodsId="+seckillGoodsId);
    }
});

File: pinyougou_seckill_interface/src/main/java/com/pinyougou/seckill/service/SeckillService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;

import java.util.List;

public interface SeckillService {

    //秒杀商品展示:
    public List<TbSeckillGoods> displaySeckillGoods();

    TbSeckillGoods findOne(Long seckillGoodsId);

    void createOrder(String userId, Long seckillGoodsId);

}
